docs/codepush.md

```diff
@@ -0,0 +1,122 @@
+# CodePush for React Native support
+
+## Overview
+
+Enable React Native Crash Reporting and Error Tracking to get comprehensive crash reports and error trends with Real User Monitoring.
+
+Each time you release a new [CodePush][1] version for your React Native application, you need to upload the source maps to Datadog to unminify errors.
+
+To achieve this, we recommand using `@datadog/mobile-react-native-code-push` in your app and the [datadog-ci][3] `react-native codepush` command to upload your source maps. It will ensure that the version will be consistent in both reported crashes and uploaded source maps.
+
+## Setup
+
+Start by following the [installation steps][2] for installing `@datadog/mobile-react-native`.
+
+Install `@datadog/mobile-react-native-code-push`.
+
+To install with NPM, run:
+
+```sh
+npm install @datadog/mobile-react-native-code-push
+```
+
+To install with Yarn, run:
+
+```sh
+yarn add @datadog/mobile-react-native-code-push
+```
+
+Replace `DdSdkReactNative.initialize` by `DatadogCodepush.initialize` in your code:
+
+```js
+import { DdSdkReactNativeConfiguration } from '@datadog/mobile-react-native';
+import { DatadogCodepush } from '@datadog/mobile-react-native-code-push';
+
+const config = new DdSdkReactNativeConfiguration(
+    '<CLIENT_TOKEN>',
+    '<ENVIRONMENT_NAME>',
+    '<RUM_APPLICATION_ID>',
+    true, // track User interactions (e.g.: Tap on buttons. You can use 'accessibilityLabel' element property to give tap action the name, otherwise element type will be reported)
+    true, // track XHR Resources
+    true // track Errors
+);
+
+await DatadogCodepush.initialize(config);
+```
+
+## Upload CodePush source maps
+
+Install [`@datadog/datadog-ci`][3] as a development dependency to your project.
+
+To install it with npm:
+
+```sh
+npm install @datadog/datadog-ci --save-dev
+```
+
+To install it with yarn:
+
+```sh
+yarn add -D @datadog/datadog-ci
+```
+
+Create an encrypted or gitignored `datadog-ci.json` file at the root of your project containing your API key and the Datadog site (if not `datadoghq.com`):
```

Comment:
maybe we should just say gitignored? to not advise pushing secrets to the repo, even in encrypted format.